packages/nc-gui/components/smartsheet/Grid.vue

```diff
@@ -462,17 +460,13 @@ reloadViewDataHook.trigger()
                     :key="columnObj.id"
                     class="cell relative cursor-pointer nc-grid-cell"
                     :class="{
-                      active:
-                        (isUIAllowed('xcDatatableEditable') && selected.col === colIndex && selected.row === rowIndex) ||
-                        (isUIAllowed('xcDatatableEditable') && selectedRange(rowIndex, colIndex)),
+                      active: isUIAllowed('xcDatatableEditable') && selected.col === colIndex && selected.row === rowIndex,
```

Comment:
I think we can replace all the `isUIAllowed('xcDatatableEditable')` calls with the computed property on top of this file called `hasEditPermission`